Src/Application/Services/MaritalStatusScoreStrategy.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Application.Common.Interfaces;
+using Application.DTOs.CustomerService;
+using Application.DTOs.MainScore;
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Application.Services;
+
+public class MaritalStatusScoreStrategy : IScoreStrategy
+{
+    public async Task<ScoreDataDto> CalculateScore(ReceivedCustomerInfoDto input,
+        CancellationToken cancellationToken = default)
+    {
+        if (IsEmpty(input))
+        {
+            return NoRecordResult();
+        }
+        //ToDO We need to implement this point calculator
+        ReceivedMaritalDto receivedMaritalDto = input.Marital;
+        return new ScoreDataDto("Score For Martial Status", 5);
+    }
+
+    private bool IsEmpty(ReceivedCustomerInfoDto input)
+    {
+        if (input is null)
+        {
+            return true;
+        }
+
```

Comment:
We can combine input and input.marital checking conditions using logical or( | ) @MehranLabour